src/main/webapp/js/email-browser.js

```diff
@@ -55,6 +55,23 @@ var jasig = jasig || {};
                 checkboxCounter = 0;
                 return checkboxCounter;
 	}
+        /**
+         * Helper function to change a role value to another
+         * @param {string} elem The ID or Class of the element which have the role you need to change
+         * @param {string} actualRole The value of role you need to change
+         * @param {string} newRole The value of role you need elem to have
+         * Used to patch fluid pager which add role="application" to allow user to use the standard keyboard commands on fluid pager.
+         *    Issue ref: FLUID-6064 https://issues.fluidproject.org/browse/FLUID-6064
+         *    Additional notes: https://www.marcozehe.de/2012/02/06/if-you-use-the-wai-aria-role-application-please-do-so-wisely/
+         *    Additional notes: http://tink.uk/using-the-aria-application-role/
+         * @supported So far tested in Chrome, FF, Safari, IE9, edge. IOS, Android.
+         */
+        function overrideA11nRole (elem, actualRole, newRole) {
```

Comment:
This patch looks okay for now.
Later this week I'll look into getting a PR into fluid infusion itself, so the fix could be a version change of fluid instead of DOM replacement in uPortal.